compose.yaml

```diff
@@ -1,62 +1,82 @@
 services:
   db:
-    image: postgres:16
-    environment:
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: password
-      POSTGRES_DB: rollupsdb
-    ports:
-      - 5433:5432
-    volumes:
-      - ./rollupsdb-dump.sql:/docker-entrypoint-initdb.d/rollupsdb-dump.sql
-  db-espresso:
-    image: postgres:16
+    image: postgres:17-alpine
+    container_name: db
     environment:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: password
       POSTGRES_DB: rollupsdb
     ports:
       - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready", "-U", "postgres", "-d", "rollupsdb"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
   devnet:
     container_name: devnet
     build:
-      context: ./rollups-node/test/devnet
+      context: ./ci/devnet
       dockerfile: Dockerfile
     ports:
       - 8545:8545
-    volumes:
-      - ./rollups-node/deployment.json:/usr/share/devnet/deployment.json
-      - ./rollups-node/anvil_state.json:/usr/share/devnet/anvil_state.json
   espresso-dev-node:
-    image: ghcr.io/espressosystems/espresso-sequencer/espresso-dev-node:20241120-patch6
+    container_name: espresso-dev-node
+    image: ghcr.io/espressosystems/espresso-sequencer/espresso-dev-node:20250428-dev-node-decaf-pos
+    ports:
+      - 24000:24000
+      - 23000:23000
+      - 20000:20000
+      # - 5432:5432
     environment:
-      ESPRESSO_SEQUENCER_L1_PROVIDER: http://devnet:8545/
-      ESPRESSO_SEQUENCER_API_PORT: 24000
-      ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: 1s
-      ESPRESSO_STATE_PROVER_UPDATE_INTERVAL: 1s
-      ESPRESSO_SEQUENCER_POSTGRES_HOST: db-espresso
-      ESPRESSO_SEQUENCER_POSTGRES_PORT: 5432
-      ESPRESSO_SEQUENCER_POSTGRES_DATABASE: rollupsdb
-      ESPRESSO_SEQUENCER_POSTGRES_USER: postgres
-      ESPRESSO_SEQUENCER_POSTGRES_PASSWORD: password
+      - ESPRESSO_BUILDER_PORT=23000
+      - ESPRESSO_DEPLOYER_ACCOUNT_INDEX
+      - ESPRESSO_DEV_NODE_PORT=20000
+      - ESPRESSO_SEQUENCER_API_PORT=24000
+      - ESPRESSO_SEQUENCER_ETH_MNEMONIC
+      - ESPRESSO_SEQUENCER_L1_PROVIDER=http://host.docker.internal:8545
+      - ESPRESSO_SEQUENCER_DATABASE_MAX_CONNECTIONS=25
+      - ESPRESSO_SEQUENCER_STORAGE_PATH=/data/espresso
+      - RUST_LOG=info
+      - RUST_LOG_FORMAT
+    volumes:
+      - espresso_storage:/data/espresso
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
     depends_on:
-      - db-espresso
       - devnet
-    ports:
-      - 24000:24000
-  c_espresso:
-    container_name: c_espresso
+    healthcheck:
+      interval: 1m30s
+      timeout: 30s
+      retries: 5
+      start_period: 30s
+    # healthcheck:
+    #   test: ["CMD", "wget", "--spider", "-q", "http://espresso-dev-node:24000/status/block-height"]
+  cartesi_espresso:
```

Comment:
Can we please use `node_espresso`..?

Reply:
I prefer a name that includes cartesi to make it easier for newcomers to understand that it is the Cartesi software.

Reply:
But the name doesn't help you know that it's the node, the place where you send tx's, do queries, etc.
The node calls itself just `node` in its docker compose.. 
What do you think about `cartesi_node_espresso`?

Reply:
Good name! it avoids the confusion of `espresso_dev_node` and `node_espresso`.

Reply:
For sure!